server/app/views/index.scala.html

```diff
@@ -1,9 +1,15 @@
-@(message: String)
+@(message: String, img: String)
```

Comment:
`imgUrl` may be a more accurate name so that a future user directly knows that she should put a url instead of say image title.

Reply:
Ok changed the name: 
9d8a79e